clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -521,4 +521,33 @@ def GlobalCtorAttr : CIR_Attr<"GlobalCtor", "globalCtor"> {
   ];
   let skipDefaultBuilders = 1;
 }
+
+def BitfieldInfoAttr : CIR_Attr<"BitfieldInfo", "bitfieldInfo"> {
```

Comment:
Let's move away from camel case for things that print CIR. Name should be `bitfield_info` or similar.